ironfish/src/mining/webhooks/webhookNotifier.ts

```diff
@@ -78,6 +79,31 @@ export abstract class WebhookNotifier {
     )
   }
 
+  poolPayoutStarted(
+    payoutId: number,
+    receives: { publicAddress: string; amount: string; memo: string }[],
+    totalShareCount: number,
+  ): void {
+    const total = receives.reduce((m, c) => BigInt(c.amount) + m, BigInt(0))
+
+    this.sendText(
+      `Creating payout of ${totalShareCount} shares to ${
+        receives.length
+      } users for ${displayIronAmountWithCurrency(
+        Number(oreToIron(Number(total.toString()))),
+        false,
+      )}(${payoutId})`,
+    )
+  }
```

Comment:
Hopefully this can be deleted later too when the pool is more reliable, so we can only have 1 notification per transaction created.